app/lib/url.rb

```diff
@@ -5,7 +5,7 @@ def self.protocol
   end
 
   def self.domain
-    if Rails.application&.initialized? && Settings::General.respond_to?(:app_domain)
+    if ActiveRecord::Base.connected?
```

Comment:
Why active record connected, and not app initialized as before? 

Asset precompilation (in docker build/buildkite, and in travis) initializes the app, so we'll call URL.url (from the liquid tag initializer) whether we have a db or not.

Since we want "live" settings from the db in production when initializing the liquid tags, we can't ask, in an initializer, if we're initialized (that answer is always false), but don't want to add a database in docker just to support assets precompilation in the builder. 

So here, when we don't have a database to check, fallback to the environment, otherwise, whenever possible, load the domain set in the settings table (which has a built in fallback to the env var), and use that value.